astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -1717,11 +1629,33 @@ Returns the Claims about the authenticated End-User.
     ```
     Authorization: Bearer [access_token]
     ```
+
+    <Aside type="caution">
+      In version `1.50.0` and later, the `aud` claim is required for all tokens. The `aud` claim value, or the first value in the case the claim contains a list, must correspond to an active application.
+
+      If you have a token that was not obtained from an OAuth flow and does not contain the `aud` claim, consider adding the claim or using the [JWT Validate API](/docs/apis/jwt#validate-a-jwt) instead.
+    </Aside>
+
+    <Aside type="note">
+      In version `1.50.0` and later when the application's <InlineField>oauthConfiguration.scopeHandlingPolicy</InlineField> is set to `Strict`, the provided token must:
+
+      * Be an access token rather than an id token
+      * Contain the `openid` scope in the `scope` claim
+
+      in order to be used with this endpoint.
+    </Aside>
   </APIField>
 </APIBlock>
 
 ### Response
 
+<Aside type="note">
+  In version `1.50.0` and later the UserInfo response changes based on the <InlineField>oauthConfiguration.scopeHandlingPolicy</InlineField> value of the application object.
+
+  * In `Strict` mode the response claims are populated based on the `scope` values of the provided token and available information on the User object. This behavior more closely aligns with specification recommendations.
+  * In `Compatibility` mode the response claims are populated based on available information on the User object only and include some claims from the provided token. This is legacy behavior maintained for backwards compatibility.
```

Comment:
This list gets repeated a lot, maybe worth extracting to a component?

Reply:
What list are you referring to?

Reply:
The list of options for the "Scope handling policy"

Reply:
I was confused because you called it out in the one place where the wording is slightly different than the others.

How far down the component rabbit hole do we want to go? I don't see other cases where enum options are split into their own component. I think that Astro is much easier to parse than ASCIIdoc, but we had some issues with includes making it hard to follow.

If we add a component for these options, what about `Relationship`, `Consent mode`, `Unknown scope policy`, etc.?

Reply:
Yeah, it's a balance. I just know when I was making suggestions, I had to change the wording in like 8 places. 

I know I created components for some lists ([JWT algorithm](https://github.com/FusionAuth/fusionauth-site/blob/master/astro/src/content/docs/apis/_jwt_algorithm.mdx)) but we haven't been consistent.

Let's leave it be for now.